packages/react-composites/src/composites/ChatComposite/FileDownloadCards.tsx

```diff
@@ -50,18 +67,27 @@ export const FileDownloadCards = (props: FileDownloadCards): JSX.Element => {
             fileExtension={extension(file.name)}
             actionIcon={<DownloadIconTrampoline />}
             actionHandler={() => {
-              props.downloadHandler && props.downloadHandler(userId, file);
+              if (props.downloadHandler) {
+                // @conditional-compile-remove(file-sharing)
+                setSpinner(true);
+                props
+                  .downloadHandler(userId, file)
```

Comment:
Please take a look at the downloadHandler function type. We are not expecting it to throw errors. The behavior of UI should depend on it's return type.